brave/brave-core
# Update bitFlyer onboarding asset

Resolves https://github.com/brave/brave-browser/issues/18001

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

- Clean profile
- Start brave with `--rewards=countryid=19024` flag, to start with bitFlyer-appropriate `countryid`
- Enable Rewards
- Verify that new bitFlyer image appears at end of onboarding flow
- Verify that clicking new bitFlyer image opens a new page to https://twitter.com/bravesoftwarejp
